robottelo/ui/rhai.py

```diff
@@ -0,0 +1,11 @@
+# -*- encoding: utf-8 -*-
```

Comment:
I created the module in order to put things at the right place, going forward the module will grow with required methods.